examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino

```diff
@@ -0,0 +1,30 @@
+/*

```

Comment:
One example folder can contain only one *.ino, has to have same name.

Reply:
Can i have pitches.h and themes.h in that folder?

Reply:
Yes, you can.